Created Instructions for Installing Developer Snapshots

wesdevpro opened the pull request:
<!-- Thank you for helping Buefy! -->

Fixes #169
<!-- If this PR fixes GitHub issue 1234, add "Fixes #1234" to the next line. -->

## Proposed Changes

- Add instructions for installing developer snapshots from our GitHub npm registry in the `README`

kikuomax commented on README.md:
````diff
@@ -114,6 +114,42 @@ If you want to customize the icons or the theme, refer to the [customization sec
+```
+3. Select a Developer Release:
+    - Visit the Buefy-next package registry: https://github.com/ntohq/buefy-next/pkgs/npm/buefy-next on GitHub.
+Choose a stable developer release that suits your needs.
````
There is no "stable" developer release. How about this?
> Choose any developer release that suits your needs.

And after that sentence, explain how the package names look like: `@ntohq/buefy-next@<package version>-<dev commit hash>`

`<package version>` is the intended release version that will include developer release versions starting with it; e.g., a developer release `@ntohq/buefy-next@0.1.3-commithash` will be included in the release `@ntohq/buefy-next@0.1.3`.

wesdevpro replied:
> There is no "stable" developer release. How about this?
> 
> > Choose any developer release that suits your needs.
> 
> And after that sentence, explain how the package names look like: `@ntohq/buefy-next@<package version>-<dev commit hash>`
> 
> `<package version>` is the intended release version that will include developer release versions starting with it; e.g., a developer release `@ntohq/buefy-next@0.1.3-commithash` will be included in the release `@ntohq/buefy-next@0.1.3`.

That's fair. These changes sound good to me. I will make those changes when I get some free time

kikuomax commented on README.md:
````diff
@@ -114,6 +114,42 @@ If you want to customize the icons or the theme, refer to the [customization sec
 ```
 
+
+### Installing Developer Snapshots from GitHub npm Registry to Access the Latest Features
````
Do we call packages in GitHub npm Registry "developer releases"? If so, please replace "developer snapshot(s)" with "developer release(s)".

wesdevpro replied:
> Do we call packages in GitHub npm Registry "developer releases"? If so, please replace "developer snapshot(s)" with "developer release(s)".

Regarding the npm packages for `Buefy-next`, I want to clarify that they are technically packages of each release made to the dev branch on our GitHub npm registry. These are not developer snapshots, but rather developer releases. 

Thank you for bringing this to my attention! I will make the necessary changes when I have some free time.

kikuomax commented on README.md:
````diff
@@ -114,6 +114,42 @@ If you want to customize the icons or the theme, refer to the [customization sec
+ USERNAME: <their GitHub username>
+ PASSWORD: <personal access token issued at Step 1>
+echo "@ntohq:registry=https://npm.pkg.github.com" > .npmrc
+```
````
It would look better structured if you indent the code snippet and remove the itemization of the sentence "In your terminal...":

```
2. Configure npm:
    
   In your terminal, execute the following commands:

    ```bash
    npm login --auth-type=legacy --registry=https://npm.pkg.github.com
     USERNAME: <their GitHub username>
     PASSWORD: <personal access token issued at Step 1>
    echo "@ntohq:registry=https://npm.pkg.github.com" > .npmrc
    ```
```

2. Configure npm:
    
    In your terminal, execute the following commands:

    ```bash
    npm login --auth-type=legacy --registry=https://npm.pkg.github.com
     USERNAME: <their GitHub username>
     PASSWORD: <personal access token issued at Step 1>
    echo "@ntohq:registry=https://npm.pkg.github.com" > .npmrc
    ```

vs

2. Configure npm:
    - In your terminal, execute the following commands:
```bash
npm login --auth-type=legacy --registry=https://npm.pkg.github.com
 USERNAME: <their GitHub username>
 PASSWORD: <personal access token issued at Step 1>
echo "@ntohq:registry=https://npm.pkg.github.com" > .npmrc
```

kikuomax commented on README.md:
````diff
@@ -114,6 +114,42 @@ If you want to customize the icons or the theme, refer to the [customization sec
+```bash
+npm install @ntohq/buefy-next@latest
+```
+**However, note it is not recommended to use the latest version of our developer snapshot, as its stability fluctuates.**
````
For better structured appearance:
```
4. Install the Package:

   Copy the provided command from GitHub, which will resemble this:

    ```bash
    npm install @ntohq/buefy-next@<package version>-<dev commit hash>
    ```

   Alternatively, to fetch the latest release from the developer snapshot registry, run:

    ```bash
    npm install @ntohq/buefy-next@latest
    ```

   **However, note it is not recommended to use the latest version of our developer snapshot, as its stability fluctuates.**
```
will yield:

4. Install the Package:

   Copy the provided command from GitHub, which will resemble this:

    ```bash
    npm install @ntohq/buefy-next@<package version>-<dev commit hash>
    ```

   Alternatively, to fetch the latest release from the developer snapshot registry, run:

    ```bash
    npm install @ntohq/buefy-next@latest
    ```

   **However, note it is not recommended to use the latest version of our developer snapshot, as its stability fluctuates.**

wesdevpro replied:
Will do! I completely forgot we can specify what language is in a code block.

kikuomax commented:
@wesdevpro Sorry for nitpicking, but I have fixed minor indentation issues.

wesdevpro commented:
> @wesdevpro Sorry for nitpicking, but I have fixed minor indentation issues.

No problem. I added these changes relatively quickly with the short amount of time that I had. This PR looks good to me
